### ISSUE TITLE
When using the keyboard, selecting an item from the Toolbar Overflow menu removes the focus from the widget

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5393]
### Bug report
When using the keyboard, selecting an item from the Toolbar Overflow menu removes the focus from the widget.

### Reproduction of the problem
1. Open the following demo: https://demos.telerik.com/kendo-ui/toolbar/keyboard-navigation
1. Navigate to the Overflow button and press Alt + Down Arrow to open the drop-down;
1. Navigate to one of the items and hit enter to execute it;

### Current behavior
The focus is removed from the Toolbar

### Expected/desired behavior
The focus should be retained within the Toolbar. After it has been closed, the Overflow menu button should remain focused.

### Environment
* **Kendo UI version:** 2019.3.1023
* **Browser:** all

**Comment by mparvanov** [comment]
The issue has been resolved since R1 2020 (2020.1.114).
